platform/cli/package.json

```diff
@@ -20,9 +20,11 @@
   "author": "OHIF Contributors",
   "license": "MIT",
   "dependencies": {
+    "@babel/core": "^7.13.10",
     "chalk": "^5.0.0",
     "commander": "^8.3.0",
     "execa": "^6.0.0",
+    "fs": "^0.0.1-security",
```

Comment:
isn't fs part of node? why we have it here? and why 0.0.1-security version?

Reply:
0.0.1-security was just part of "yarn install fs" 🤷 .

I'm a little inexperienced with node, good catch! I'll remove the dep.